patches/chrome-browser-ui-autofill-payments-save_card_bubble_controller_impl.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc b/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc
+index e576d1a0a1f331679c586ef63a17c38063501611..cff2b3f9b8e8820627819cae1421b712762f42f2 100644
+--- a/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc
++++ b/chrome/browser/ui/autofill/payments/save_card_bubble_controller_impl.cc
+@@ -437,6 +437,7 @@ bool SaveCardBubbleControllerImpl::ShouldRequestExpirationDateFromUser() const {
+ }
+ 
+ bool SaveCardBubbleControllerImpl::ShouldShowSignInPromo() const {
++  BRAVE_SHORT_CIRCUIT_TO_FALSE;
```

Comment:
@darkdh this is also controlled by ```const syncer::SyncService* sync_service =
      ProfileSyncServiceFactory::GetForProfile(GetProfile());

  return !sync_service ||
         sync_service->HasDisableReason(
             syncer::SyncService::DISABLE_REASON_NOT_SIGNED_IN) ||
         sync_service->HasDisableReason(
             syncer::SyncService::DISABLE_REASON_USER_CHOICE);```
and it seems like that might be a better place to fix this by changing to `DISABLE_REASON_NONE` or maybe with your change not disabled at all?

Reply:
Upcoming brave-syncer PR will have DISABLE_REASON_NONE so I think https://github.com/brave/brave-core/pull/2829/commits/e5068be2202ef8f738e76956e92b9a96269c5f81 handle this properly